docat/docat/app.py

```diff
@@ -151,6 +156,11 @@ def upload(
 @app.put("/api/{project}/{version}/tags/{new_tag}/", response_model=ApiResponse, status_code=status.HTTP_201_CREATED)
 def tag(project: str, version: str, new_tag: str, response: Response):
     destination = DOCAT_UPLOAD_FOLDER / project / new_tag
+    source = DOCAT_UPLOAD_FOLDER / project / version
+
+    if not source.exists():
+        response.status_code = status.HTTP_404_NOT_FOUND
+        return ApiResponse(message=f"Version {version} not found")
```

Comment:
@reglim can we change that to something like "Could not create tag {new_tag} because there is no Project Version {version}